constants/index.ts

```diff
@@ -24,6 +24,36 @@ export default Object.freeze({
   DECIMAL_PRECISION: 6,
   ORDER_PICKED: 'Order-picked-up',
   ORDER_DELIVERED: 'Order-delivered',
+  FIS_SEARCH: 'search',
```

Comment:
Pls merge the constants with same values. eg RET_SEARCH, FIS_SEARCH, MOB_SEARCH and replace it across all the files